sweepai/handlers/on_ticket.py

```diff
@@ -210,7 +210,8 @@ def on_ticket(
     payment_link = "https://buy.stripe.com/9AQ8zB26letOgzC5kp"
     user_type = "💎 Sweep Pro" if is_paying_user else "⚡ Sweep Free Trial"
     payment_message = f"{user_type}: I used {model_name} to create this ticket. You have {ticket_count} GPT-4 tickets left." + (f" For more GPT-4 tickets, visit [our payment portal.]({payment_link})" if not is_paying_user else "")
-    payment_message_start = f"{user_type}: I'm creating this ticket using {model_name} with slow mode set to {slow_mode}. You have {ticket_count} GPT-4 tickets left." + (f" For more GPT-4 tickets, visit [our payment portal.]({payment_link})" if not is_paying_user else "")
+    slow_mode_status = "using slow mode" if slow_mode else ""
+    payment_message_start = f"{user_type}: I'm creating this ticket using {model_name} {slow_mode_status}. You have {ticket_count} GPT-4 tickets left." + (f" For more GPT-4 tickets, visit [our payment portal.]({payment_link})" if not is_paying_user else "")
```

Comment:
add space only if slow_mode true

Reply:
add space before, only if slow_mode is true. Then remove the space from the second string